### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by PitchyPA** [bodymovin/bodymovin#141]
Hello,

When I'm trying to extract the zipped file on build/extension/bodymovin.zxp to the adobe CEP folder, the process stop and I have a message error. 
Apparently I have to check that the file manifest.xml is valid.

Do you have any solution for my issue ?

I use Windows and cc2015.3

**Comment by bodymovin** [comment]
hi
where does the message show up? when opening AE?

**Comment by PitchyPA** [comment]
I tried to install with Adobe Extension Manager.

We made a new try and we have several message error : 

First message : "The extension create a conflict with "com.bodymovin.bodymovin". It says it's not installed in the After Effects CC 2015.3 product.

Then the install says "impossible to find data into the data base"

But I followed instructions of this link : https://github.com/bodymovin/bodymovin
I think it's not available with the latest version After Effects...

**Comment by JackPollitt** [comment]
Hey Hernan, i'm having the same/similar problem.


Previously had it installed working perfectly, left my office for holiday and the file had been deleted. 

Don't remember having to use a zxp file before but this time around it won't let me install coming up with these errors.

<img width="358" alt="screen shot 2016-09-05 at 10 00 21" src="https://cloud.githubusercontent.com/assets/19855318/18249579/1d941e94-7377-11e6-9de1-eb0084082488.png">
<img width="351" alt="screen shot 2016-09-05 at 12 42 45" src="https://cloud.githubusercontent.com/assets/19855318/18249580/1da3bfa2-7377-11e6-9763-228f1fe7d63c.png">

Top error was before i installed the latest version of extension manager. bottom is after installed.

Any ideas/thoughts? Really would like to get this working so i can carry on SVG'ing


Thanks Hernan

Jack

**Comment by bodymovin** [comment]
Hi, it should be working with the latest version.
I'll try to reproduce the error.
But are you installing it through the add-ons store?

**Comment by JackPollitt** [comment]
Yes, my creative cloud updated this morning. AE is up to date as is the extension manager.
installing via the add ons store also.. My colleague did the same to see if it would work on his machine and it worked first time.

Left me a bit confused as to what could be wrong.

Thanks for the speedy reply as always.

Jack Rogers
Motion Designer
 
Pollitt & Partners
9–10 Floral Street
London WC2E 9HW  <http://maps.google.co.uk/maps/place?hl=en&georestrict=input_srcid%3A9be3085c7a6e454f>
T +44 (0)20 7379 6709 <tel:+442073796709>
pollittandpartners.com <http://www.pollittandpartners.com/?utm_source=email-sig-logo&utm_medium=Jack%20Rogers&utm_campaign=Email%20signature>
  <http://www.pollittandpartners.com/?utm_source=email-sig-logo&utm_medium=Jack%20Rogers&utm_campaign=Email%20signature>

**Comment by bodymovin** [comment]
can you try uninstalling the plugin and then installing it again?

**Comment by JackPollitt** [comment]
Success i’ve fixed it. Removed app, uninstalled etc. Deleted folder of bodymovin. Restarted machine, re-installed via app store and didn’t need to open the zxp. All works fine…

Now back to working out how to finish the animation! 

Thanks Hernan.

Jack Rogers
Motion Designer
 
Pollitt & Partners
9–10 Floral Street
London WC2E 9HW  <http://maps.google.co.uk/maps/place?hl=en&georestrict=input_srcid%3A9be3085c7a6e454f>
T +44 (0)20 7379 6709 <tel:+442073796709>
pollittandpartners.com <http://www.pollittandpartners.com/?utm_source=email-sig-logo&utm_medium=Jack%20Rogers&utm_campaign=Email%20signature>
  <http://www.pollittandpartners.com/?utm_source=email-sig-logo&utm_medium=Jack%20Rogers&utm_campaign=Email%20signature>

**Comment by bodymovin** [comment]
nice! :)
